README.md

```diff
@@ -48,4 +48,4 @@ $ kubectl get no -o json | jq '.items[].metadata.labels'
 
 ```
 
-[documentation]: https://kubernetes-sigs.github.io/node-feature-discovery
+[documentation]: https://kubernetes-sigs.github.io/node-feature-discovery/v0.16
```

Comment:
why did you change this? Don't you think that it's better to redirect to a stable version?

Reply:
yeah is the script that change that bit... grrrr one second

Reply:
done